### ISSUE TITLE
Need some indication when -t overrides -d and -T overrides -D

### ISSUE THREAD

**Issue by harelba** [harelba/q#119]
The -t shorthand parameters override specific -d delimiter setting without issuing any indication. This might lead to confusion. Same for output delimiters.

A warning needs to be added in such case, or even failing to run the command.

**Comment by marcobra** [comment]
I agree i spent about an hour to get a pipe delimited file working was the -t param was the issue 
it kill the -d delimiter param with no warning or doc about it:

q -d "|" -H "select cnum,canno,de_nome,de_recapiti,impo_2016 from ./myarchive2016.psv where de_nome like '%scu%' limit 3"

works
